app/views/layouts/_header.html.erb

```diff
@@ -11,11 +11,23 @@
       </div>
       <div class="pull-right auto-width-right">
         <ul class="top-details menu-beta l-inline">
-          <li><a href="#"><i class="fa fa-user"></i>
-            <%= t "header.account" %></a>
+          <% if current_user.present? %>
+          <li>
+            <a href="javascript:void(0)">
+              <%= t("header.hello") + current_user.full_name %>
```

Comment:
chỗ này dùng cách này nhé passing variables to translations
http://guides.rubyonrails.org/i18n.html#passing-variables-to-translations